util/config_test.go

```diff
@@ -201,7 +202,10 @@ func TestReadConfig(t *testing.T) {
 	if fconfig == nil {
 		t.Error("Empty string ReadConfig() should result in a non-nil config")
 	}
-	fconfig.String()
+
+	if strings.TrimSpace(fconfig.String()) == "" {
+		t.Error("Empty string")
+	}
```

Comment:
Like the one before, the output of this can never be `""`. Unlike the one before, this function can never fail as long as fconfig is not nil, which is checked the line before. 

I think it exists just to get unit test coverage over those lines

Reply:
replaced with `_ = ...`, even if it doesn't test anything...